### ISSUE TITLE
Support given_name and family_name, add it to the "handful" of supported Standard claims 

### ISSUE THREAD

**Issue by edeati** [dexidp/dex#2358]
### Preflight Checklist

- [X] I agree to follow the [Code of Conduct](https://github.com/dexidp/dex/blob/master/.github/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://www.github.com/dexidp/dex/issues) for an issue that matches the one I want to file, without success.

### Problem Description

`given_name` and `family_name` Standard claims are not included in the ID Token when you request `profile` scope. 

These are fairly  standard claims and applications would be using these claims to display the user's name.

### Proposed Solution

Include family_name and given_name in the ID token
```
  "claims_supported": [
    "iss",
    "sub",
    "aud",
    "iat",
    "exp",
    "email",
    "email_verified",
    "locale",
    "name",
    "preferred_username",
    "at_hash",
    "given_name",
    "family_name"
  ]
```

**Comment by sagikazarmark** [comment]
@edeati Yeah, these are standard claims....for OIDC. Unfortunately a bunch of providers (SAML, LDAP, OAuth2) don't necessarily provide the same information, hence we've been reluctant to include these claims.

However, it might make sense to revisit this decision.

**Comment by Udbv** [comment]
I would suggest to make it optional, per each connector. ideally there should be a mapping for each connector.
For example LDAP:

- given_name: custom_attr_1
- family_name: custom_attr_2.

And for me this custom_attr_XX are base64 encoded(non latin characters)

**Comment by yacut** [comment]
any update?

**Comment by cldfzn** [comment]
Is there a workaround for this?
